lib/g_searcher_web/controllers/report_controller.ex

```diff
@@ -0,0 +1,28 @@
+defmodule GSearcherWeb.ReportController do
+  use GSearcherWeb, :controller
+
+  alias GSearcher.Reports
+  alias GSearcherWeb.DashboardView
+
+  def create(conn, %{"report" => report_params}) do
+    user = conn.assigns.current_user
+
+    with %{"title" => title, "csv" => file} <- report_params,
```

Comment:
What happen if the report params is invalid, no title or csv key 💭

Reply:
I validate the report params more in-depth in #6 👍